src/utils/network/networking.ts

```diff
@@ -16,6 +16,15 @@ export async function httpGet(url: string, options?: AxiosRequestConfig): Promis
   return response.data ? response.data : null;
 }
 
+export async function httpGetWithHeaders(url: string, options?: AxiosRequestConfig): Promise<any> {
+  const headers = options?.headers as RawAxiosRequestHeaders | undefined;
+  const response: AxiosResponse = await axios.get(url, {
+    ...options,
+    headers: { ...headers, Pragma: 'no-cache' },
+  });
+  return response.data ? response : null;
+}
```

Comment:
Not sure I like the name all that much.. It's not 'with headers', it's with the whole response object. Frankly I don't like the way we usually pluck out the data from the responses, so if I'm not sure if just renaming it to `httpGetRaw()` would be better, or if we should just stop wrapping axios entirely. Anyway, it's all fine for v3, no need to be picky there.

Reply:
Yeah, but if you're renaming all usages you might as well just use `httpGet(...)?.data` and just have one function to rule them all. The same then goes for all the `http*` functions, so I'd prefer to clean it all up or leave it be.. 😊